From: Wolfgang Rosenauer <stark@suse.de>
Subject: fix IDL installation dir 
This is only needed to get the correct path in xulrunner-config

Index: config/autoconf.mk.in
===================================================================
RCS file: /cvsroot/mozilla/config/autoconf.mk.in,v
retrieving revision 3.449
diff -u -r3.449 autoconf.mk.in
--- config/autoconf.mk.in	16 Feb 2008 22:23:08 -0000	3.449
+++ b/config/autoconf.mk.in	19 Feb 2008 14:07:13 -0000
@@ -62,7 +62,7 @@
 libdir		= @libdir@
 datadir		= @datadir@
 mandir		= @mandir@
-idldir		= $(datadir)/idl/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
+idldir		= $(includedir)
 
 installdir	= $(libdir)/$(MOZ_APP_NAME)-$(MOZ_APP_VERSION)
 sdkdir		= $(libdir)/$(MOZ_APP_NAME)-devel-$(MOZ_APP_VERSION)
